test/integration/Datatoken.test.ts

```diff
@@ -169,7 +170,7 @@ describe('Datatoken tests', async () => {
       method: 'POST',
       body: JSON.stringify(initialQuery)
     })
-    await sleep(2000)
+    await sleep(3000)
```

Comment:
Just checking... is it really necessary to increase all of the sleep time to get this test to pass?